.travis.yml

```diff
@@ -3,8 +3,22 @@ language: java
 jdk:
   - oraclejdk8
 
+env:
+  - NODE_VERSION="5.12.0"
+
+cache:
+  directories:
+    - node_modules
+
+before_install:
+  - nvm install $NODE_VERSION
+
+install:
+  - npm install
+
 after_success:
   - mvn clean test jacoco:report coveralls:report
```

Comment:
I think this step can just be `mvn test jacoco:report coveralls:report` and it won't have to reinstall all the dependencies again.